Makefile

```diff
@@ -122,7 +122,10 @@ docker-push:
 	docker push ${IMG}
 
 e2e-setup:
-	bin/vela install --set installCertManager=true --image-pull-policy IfNotPresent --image-repo vela-core-test --image-tag $(GIT_COMMIT)
+	helm repo add jetstack https://charts.jetstack.io
+	helm repo update
+	helm upgrade --install --create-namespace --namespace cert-manager cert-manager jetstack/cert-manager --version v1.2.0  --set installCRDs=true --wait
+	helm upgrade --install --create-namespace --namespace vela-system --set image.pullPolicy=IfNotPresent --set admissionWebhooks.certManager.enabled=true --set image.repository=vela-core-test --set image.tag=$(GIT_COMMIT) --wait kubevela ./charts/vela-core
```

Comment:
why use `helm upgrade` instead of `helm install`

Reply:
Sorry for the late reply, was on vacation :-) Running helm upgrade with the option --install will do a normal install if the release is missing, if it's already existing, it will upgrade it. This comes handy when you test and run in locally and also works fine in a fresh ci env.

Reply:
cool, that's good